README.md

```diff
@@ -16,8 +16,36 @@ truthtable reads in a logical expression and generates from that a LaTeX tabular
 For more examples, look into the [example folder](../master/example).
 
 ## Input Syntax
-
-¯\\_(ツ)_/¯
+The program supports a number of logical operators and their notation:
+- disjunction
+  - `or`, `|`, `\or`, `\vee`, `\lor`
+- conjunction
+  - `and`, `&`, `\and`, `\wedge`, `\land`
+- negation
+  - `not`, `!`, `\neg`, `\lnot`
+- xor
+  - `xor`, `^`, `\oplus`, `\veebar`
+- implication
+  - `>`, `\imp`, `\rightarrow`, `\implies`
+- equivalence
+  - `=`, `\aeq`, `\leftrightarrow`
+  
+Please note, that the representation you input will be used in the output.
```

Comment:
Please change to `Please note that the output will use the same token representation as your input.`